### ISSUE TITLE
Documentation regarding setting up SageMath's Jupyter kernel in an existing installation points to wrong directory

### ISSUE THREAD

**Issue by behackl** [sagemath/sage#34756]
<div id="comment:0"></div>

The instructions given in https://doc.sagemath.org/html/en/installation/launching.html#setting-up-sagemath-as-a-jupyter-kernel-in-an-existing-jupyter-notebook-or-jupyterlab-installation refer to a command "along the lines" of

```
ln -s $(sage -sh -c 'ls -d $SAGE_VENV/share/jupyter/kernels/sagemath') $HOME/.local/share/jupyter/sagemath-dev
```
to make the SageMath kernel available to an existing (system) Jupyter installation. The actual kernel directory is (cf. https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernel-specs) in `$HOME/.local/share/jupyter/kernels`, so `/kernels` should be appended to the respective lines.

Component: **documentation**

Keywords: **easy**

Author: **Benjamin Hackl**

Branch/Commit: **[`096e228`](https://github.com/sagemath/sagetrac-mirror/commit/096e22811a936af07c4b3b55974ca30f02e7360d)**

Reviewer: **Matthias Koeppe**

_Issue created by migration from https://trac.sagemath.org/ticket/34756_

**Comment by behackl** [comment]
Branch: **[u/behackl/docs/fix-jupyter-kernel-dir](https://github.com/sagemath/sagetrac-mirror/tree/u/behackl/docs/fix-jupyter-kernel-dir)**

**Comment by behackl** [comment]
Commit: **[`096e228`](https://github.com/sagemath/sagetrac-mirror/commit/096e22811a936af07c4b3b55974ca30f02e7360d)**

**Comment by behackl** [comment]
<div id="comment:2"></div>

New commits:
<table><tr><td><a href="https://github.com/sagemath/sagetrac-mirror/commit/096e22811a936af07c4b3b55974ca30f02e7360d"><code>096e228</code></a></td><td><code>trac 34756: appended /kernels to jupyter kernel paths</code></td></tr></table>

**Comment by behackl** [comment]
Author: **Benjamin Hackl**

**Comment by mkoeppe** [comment]
Reviewer: **Matthias Koeppe**

**Comment by mkoeppe** [comment]
<div id="comment:3" align="right">comment:3</div>

Thanks for catching & fixing this mistake

**Comment by vbraun** [comment]
Changed branch from **[u/behackl/docs/fix-jupyter-kernel-dir](https://github.com/sagemath/sagetrac-mirror/tree/u/behackl/docs/fix-jupyter-kernel-dir)** to **[`096e228`](https://github.com/sagemath/sagetrac-mirror/commit/096e22811a936af07c4b3b55974ca30f02e7360d)**
